modules/nw-ingress-sharding-route-configuration.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * configuring_ingress_cluster_traffic/configuring-in4ress-cluster-traffic-ingress-controller.adoc
+// * networking/routes/route-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-ingress-sharding-route-configuration_{context}"]
+= Creating a route for  IngressController sharding
+
+A route allows you to host your application at a particular URL. The following procedure describes how to create a route for Ingress Controller sharding, using the `hello-openshift` application as an example.
+
+Ingress Controller sharding is useful when balancing incoming traffic load among
+a set of Ingress Controllers and when isolating traffic to a specific Ingress
+Controller. For example, company A goes to one Ingress Controller and company B
+to another.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You are logged in as an administrator.
```

Comment:
I agree it should be more specific, I'll go ahead and change it here. Changing it in other places is outside the scope of this PR, but it is a good question and something I'll look into since it's pretty widespread